### DISCUSSION TITLE
Probleme installation

### DISCUSSION THREAD

**Discussion by sououba** [Nash115/frenchATC-for-MSFS2020#29] [Help] [answered]
Bonjour, lorsque je lance libraries_installer.bat même en admin il ne reconnait pas 'pip', et si je lance atc.py il me dit qu'il faut lancer libraries_installer.bat, une solution svp ? merci beaucoup
![ATC](https://github.com/Nash115/frenchATC-for-MSFS2020/assets/144865408/4ec1ec2c-1879-4626-bae4-cdf4a006f4e2)
![atc 2](https://github.com/Nash115/frenchATC-for-MSFS2020/assets/144865408/23960ef1-e2d3-4d14-8866-c62ec1613d5c)

**Comment by Nash115** [comment, marked as answer]
Bonjour, je pense savoir d'où vient le problème.
Durant votre installation python avez vous bien ajouté python.exe au "PATH" ?
[cliquez ici pour plus d'infos](https://github.com/Nash115/frenchATC-for-MSFS2020/wiki/Aide-pour-le-t%C3%A9l%C3%A9chargement,-l'installation-et-l'utilisation-du-mod#important--durant-linstallation-il-est-indispensable-dactiver-cela-)
Python est bien installé sur votre machine, cependant vous pouvez relancer le/un fichier d'installation de python et le réinstaller afin d'activer cette option...

Car en effet si cette option n'est pas activée il vous sera impossible d'installer les librairies
